client/me/controller.js

```diff
@@ -192,15 +193,17 @@ export default {
 		analytics.pageView.record( basePath, ANALYTICS_PAGE_TITLE + ' > Notifications' );
 
 		React.render(
-			React.createElement( NotificationsComponent,
-				{
-					user: user,
-					userSettings: userSettings,
-					blogs: sites,
-					devices: devices,
-					path: context.path
-				}
-			),
+			React.createElement( Provider, { store: context.store }, () => {
```

Comment:
> But single point of render would be awesome

FWIW, I was having a play with doing this with the existing `page.js` setup  in https://github.com/Automattic/wp-calypso/pull/1186